packages/visx-annotation/src/components/Label.tsx

```diff
@@ -67,7 +69,78 @@ function getCompletePadding(padding: LabelProps['backgroundPadding']) {
   return { ...DEFAULT_PADDING, ...padding };
 }
 
-export default function Label({
+export default function Label({ ...props }: LabelProps) {
```

Comment:
I think this has a unnecessary clone / spread

```suggestion
export default function Label(props: LabelProps) {
```